ui/src/main/java/com/wiselab/ui/Buttons.kt

```diff
@@ -0,0 +1,154 @@
+package com.wiselab.ui
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+object Buttons {
```

Comment:
Since most of the parameters are the same, could it also be an option to introduce a buttons class or function with all the parameters and then an extra `ButtonType` enum? 

Not sure if this a good idea or not 😄

Reply:
This is fine. If a composable is a wrapper for an other composable (mostly used in theming), you should expose every (or as much as possible) of the wrapped composable to keep flexibility.  In some other cases like different types of item cards or something like that, you can use Type enums. But in this case where it is theming in a core library, we should stay as flexible as possible. It looks cumbersome seeing the same parameters over and over again. But when you need to change one parameter that is blocked off by the wrapper, there is a problem. The padding on textfields by material is the biggest example of this 😄